brigade-controller/cmd/brigade-controller/controller/handler.go

```diff
@@ -234,7 +234,7 @@ func workerEnv(project, build *v1.Secret, config *Config) []v1.EnvVar {
 		{Name: "BRIGADE_REMOTE_URL", Value: string(project.Data["cloneURL"])},
 		{Name: "BRIGADE_WORKSPACE", Value: "/vcs"},
 		{Name: "BRIGADE_PROJECT_NAMESPACE", Value: build.Namespace},
-		{Name: "BRIGADE_SERVICE_ACCOUNT", Value: config.WorkerServiceAccount},
+		{Name: "BRIGADE_SERVICE_ACCOUNT", Value: config.JobServiceAccount},
```

Comment:
If we consider per-project service accounts, the value from the project secret would take precedence over the default one set by the controller, right?